core-services/prow/03_deployment/admin_config_updater_rbac.yaml

```diff
@@ -6,6 +6,18 @@ items:
   metadata:
     name: config-updater
     namespace: ci
+- kind: ClusterRoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1beta1
+  metadata:
+    name: ci-admins-config-updater
+  roleRef:
+    kind: ClusterRole
+    apiGroup: rbac.authorization.k8s.io
+    name: sudoer
```

Comment:
For audit in the future we may just want to give the SA itself a higher level of privilege so it can do things without impersonation, or we'll just see system:admin on every action